### DISCUSSION TITLE
Keep the 18+ tag when you disable the NSFW filter

### DISCUSSION THREAD

**Discussion by edgartaor** [civitai/civitai#244] [Ideas]
![image](https://user-images.githubusercontent.com/430438/218286068-8f85720c-c1a3-4c0a-95ea-3f31d5d90803.png)![image](https://user-images.githubusercontent.com/430438/218286039-46c61278-0e0f-460c-b570-054ee374d2fb.png)

That's useful info while browsing the webpage.

**Comment by JustMaier** [comment]
We have it setup this way for Site Mods actually. We were sure if it'd be annoying or not for users.
If more people express an interest in this we can certainly make this adjustment.

Would you just want to see it on the model card or on all the images?

**Reply by edgartaor** [reply]
On the model card is where would be more useful.

Here is another submit similar to this one: https://github.com/orgs/civitai/discussions/237
>I don't want to hide NSFW, but I want to know if it is NSFW.

**Reply by JustMaier** [reply]
Thanks for pointing that out

**Comment by manuelurenah** [comment]
Hey @edgartaor! This has already been implemented, you should be able to see which models are labeled as NSFW (18+) even if you have opted-out from blurring this kind of content from your account settings. Thanks again for bringing this up 🙌
